repro_diff_parser_bug.py

```
@@ -0,0 +1,85 @@
+            tmpdir, "café.md", "line1\n", "line1\nadded line\n"
+        )
+        print("--- raw git diff ---")
+        print(diff)
+        blocks = parser.parse(
+            diff, repository="test-repo", commit_sha="abc123",
+            committed_at=datetime.now(timezone.utc),
+        )
+        print(f"--- DiffParser result: {len(blocks)} block(s) ---")
+        for b in blocks:
+            print(f"  file_path={b.file_path!r} added_lines={b.added_lines!r}")
+        print(f"EXPECTED: 1 block, file_path='café.md', added_lines=['added line']")
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)